documentation/source/c_symbols.txt

```diff
@@ -0,0 +1,63 @@
+FliCbHdl
```

Comment:
Why is this file needed? If these words are used in cross-references, aren't they excluded from the spellcheck?

Reply:
There is an option to skip such MixedCase identifiers but I explicitly disabled it because I wanted "GitHub", "SystemVerilog" etc. to be checked.
The identifiers in this file are generated by breathe for the C++ Library Reference section but apparently not marked up so that spellchecking would skip them.

Reply:
I imagine that in future we could either
* Create a custom spelling filter that reads the breathe data-files to produce this list at runtime
* Change breathe to not output a format that does this

For now, what you have is a tidy workaround - shame you can't put a comment in a wordlist file.